azerothcore/azerothcore-wotlk
# fix(Zone/Silithus): Wind stone summons

## Changes Proposed:
-  Fix npc texts showing variables
-  Fix summoned creatures to no longer be stationary
- Fix issue where you had to double click on gossip after spawning the first creature


## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9807
- Closes https://github.com/chromiecraft/chromiecraft/issues/2622

## SOURCE:


N/A 

## Tests Performed:

- Build works
- SQL works

> /* Affected rows: 12  Found rows: 5  Warnings: 0  Duration for 2 queries: 0,000 sec. */

> ========== Build: 4 succeeded, 0 failed, 15 up-to-date, 1 skipped ==========

## How to Test the Changes:


```
.go c 42955
.additem 20406
.additem 20407
.additem 20408
wear items
use [Lesser Wind Stone] near
```

Do above steps multiple times, address _changes proposed_ section and check each thing

## Known Issues and TODO List:


N/A


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
